sdk/core/azure-core/src/http/winhttp/win_http_transport.cpp

```diff
@@ -654,6 +672,9 @@ std::unique_ptr<RawResponse> WinHttpTransport::SendRequestAndGetResponse(
 
 std::unique_ptr<RawResponse> WinHttpTransport::Send(Request& request, Context const& context)
 {
+  using namespace Azure::Core::Http::_internal;
+  m_noClientCert = WinHttpTransportContextProvider::HasNoClientCertificateConfiguration(context);
```

Comment:
There is a race here if the WinHttpTransport is shared across threads.

Reply:
Fixed it.  Thank you